### PR TITLE
Display name instead of ID for bots

### PR DESCRIPTION
**Pull request by terlar**
Add Bot entity to be able to get the name

Right now all the bots are represented as their ID:s which is not very
useful, this fixes this.

This is my first python code so please let me know if something is
weird.

### CODE REVIEW

**Comment by rawdigits** [comment]
Thanks! This looks good, but going to do some testing before a merge.

**Comment by pbrisbin** [comment]
FWIW, this is working well for me

**Comment by tfheen** [comment]
Hi @terlar, any chance you could rebase this? Master's moved ahead in the meantime so this no longer merges cleanly.

**Comment by terlar** [comment]
Updated, I hope this will be the last rebase, I have tried to keep it up to date for quite some time.

File: wee_slack.py [hunk 1]
```diff
@@ -1469,7 +1496,7 @@ def get_user(message_json, server):
     elif 'service_name' in message_json:
         name = u"-{}-".format(message_json["service_name"])
     elif 'bot_id' in message_json:
-        name = u"-{}-".format(message_json["bot_id"])
+        name = u"-{}-".format(server.bots.find(message_json["bot_id"]).name)
```
Comment (tfheen) [inline wee_slack.py]: This elif stanza should be moved earlier so we prefer the bot name if one exists.

Also, I'm wondering if we should use something more distinctive than `-$Name-`. @rawdigits suggested `:]`, which kinda looks like a robot. I tried using a unicode symbol, but that leads to various warnings, so I'm somewhat leaning towards the `:]` idea too.

**Comment by tfheen** [comment]
This looks pretty good and seems to work fine in testing. I've that one comment in there, but it's a fairly minor thing. Opinions welcome. :-)

**Comment by terlar** [comment]
I have done your suggested changes, perhaps one could let that be configurable how to display the bot name. But that could be a later improvement. I have been running this since February. But with the latest rebase and changes only a few times. If something is broken it should be related to the rebase but I don't think so.

**Comment by tfheen** [comment]
This looks good to me. @rawdigits, ok to merge?

**Pull request merged**